src/main/java/net/snowflake/client/jdbc/RestRequest.java

```diff
@@ -247,17 +287,20 @@ public static CloseableHttpResponse execute(
       } catch (Exception ex) {
 
         savedEx = ex;
-        // if the request took more than 5 min (socket timeout) log an error
+        // if the request took more than socket timeout log an error
         if ((System.currentTimeMillis() - startTimePerRequest)
             > HttpUtil.getSocketTimeout().toMillis()) {
           logger.warn(
-              "HTTP request took longer than 5 min: {} sec",
-              (System.currentTimeMillis() - startTimePerRequest) / 1000);
+              "{}HTTP request took longer than socket timeout {} ms: {} ms",
+              requestIdStr,
+              HttpUtil.getSocketTimeout().toMillis(),
+              (System.currentTimeMillis() - startTimePerRequest));
```

Comment:
should we have here the same time as taken in if above? should we have an additional variable here for `System.currentTimeMillis()` created before if??